src/Octoshift/Commands/GrantMigratorRole/GrantMigratorRoleCommandBase.cs

```diff
@@ -43,7 +52,8 @@ public override GrantMigratorRoleCommandHandler BuildHandler(GrantMigratorRoleCo
 
         var log = sp.GetRequiredService<OctoLogger>();
         var githubApiFactory = sp.GetRequiredService<ITargetGithubApiFactory>();
-        var githubApi = githubApiFactory.Create(args.GhesApiUrl, args.GithubPat);
+        var api_url = args.TargetApiUrl ?? args.GhesApiUrl;
```

Comment:
all local variable names should be camel cased:

```suggestion
        var apiUrl = args.TargetApiUrl ?? args.GhesApiUrl;
```